cmd/create/cluster/cmd.go

```diff
@@ -1046,8 +1047,9 @@ func run(cmd *cobra.Command, _ []string) {
 
 		if !isHcpBillingTechPreview {
 
-			if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
-				r.Reporter.Errorf("Billing account is invalid. Run the command again with a valid billing account.")
+			err := validateBillingAccount(billingAccount)
+			if err != nil {
+				r.Reporter.Errorf("%s", err)
```

Comment:
```suggestion
				r.Reporter.Errorf("%v", err)
```

`err` is not a string right? so I think you have to use `err.Error()` or `%v`

Reply:
@mikeshng err will be a string as it's the return of validateBillingAccount in line 3260, but I'll fix the rest.